src/pages/Meeting/MeetingDetailPage.tsx

```
@@ -10,7 +11,33 @@ import { useMeetingDetail } from "../../hooks/useClubMeeting";
+    }
+  }, [navigationType, refetch, meetingId, queryClient]);
+
+  useEffect(() => {
+    const handler = (e: PageTransitionEvent) => {
+      // bfcache에서 복원되거나, 단순히 페이지 표시될 때도 갱신
+      if (e.persisted || true) {
+        refetch();
+      }
+    };
+    window.addEventListener("pageshow", handler as any);
+    return () => window.removeEventListener("pageshow", handler as any);
```

Biome (2.1.2)

[error] Unexpected constant condition.



(lint/correctness/noConstantCondition)